### ISSUE TITLE
Education Parity Edits

### ISSUE THREAD

**Issue by lunascoop** [department-of-veterans-affairs/va.gov-team#2098]
Education redirects cannot be completed until all the additional parity issues mentioned in [Issue #15512 ](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15512)are resolved. The pages below require additional parity work. The original comment/question for each page is included and its current status.  
 
VA.gov page: www.va.gov/education/about-gi-bill-benefits/how-to-use-benefits/non-college-degree-programs/
legacy page: www.benefits.va.gov/gibill/non_college_degree.asp
**Original comment:** Add barber/beautician school to VA.gov page?
**Review:** Content regarding barber/beautician school has not been added.

VA.gov: www.va.gov/education/about-gi-bill-benefits/how-to-use-benefits/test-fees/
Legacy page: www.benefits.va.gov/gibill/licensing_certification.asp
**Original comment:** Add the below info to VA.gov page?
Fees connected with obtaining a license or certification are not reimbursable.
**Review:** Verification needed that content has been updated per the above statement. According to the website, the above statement isn't entirely true as certain license and certification fees are reimbursable. There is a URL link (WEAMS) to where a user can find covered licenses and certifications and the reimbursed amount.

VA.gov page: www.va.gov/education/choosing-a-school/
Legacy page: www.benefits.va.gov/gibill/choosing_a_school.asp
**Original comment:** Add a link to the GI Bill Comparison Tool to the VA.gov page?
add links that appear at the bottom of this page (https://www.benefits.va.gov/gibill/choosing_a_school.asp) to the new VA page?
**Review:** Link to new GI Bill Comparison Tool was added via the "Find which schools we've approved" bullet point. However, there was no link to the above URL added. Need to confirm if this update is complete.

VA.gov page: www.va.gov/education/how-to-apply/
Legacy page: www.benefits.va.gov/gibill/apply.asp
**Original comment:** Add a link to VA's interactive map for checking local processing times?
**Review:** Link to VA's interactive map for checking local processing has not been added.

Va.gov page: www.va.gov/education/other-va-education-benefits/reap/
Legacy page: www.benefits.va.gov/gibill/reap.asp
**Original comment:** Add information about Colmery Act and reservists crediting their REAP eligibility toward Post-9/11 benefits?
**Review:** Information above has not been added.

Va.gov page: www.va.gov/education/other-va-education-benefits/veap/
Legacy page: www.benefits.va.gov/gibill/veap.asp
**Original comment:** Add Air Force Eligibility requirements to Va.gov page?
**Review:** Information above has not been added.

Va.gov page: www.va.gov/education/survivor-dependent-benefits/dependents-education-assistance/
Legacy page: www.benefits.va.gov/GIBILL/DEA.asp
**Original comment:** Add Additional Assistance information to the VA.gov page?
**Review:** Information above has not been added.

**Comment by Karak888** [comment]
@peggygannon can you please confirm if these items were deliberalety not included or do they still need to happen? If we chose not to bring it over, does this acctually complete parity?  Please let us know.  
CC: @jenniferlee-dsva

**Comment by peggygannon** [comment]
@Karak888 I believe these parity additions/changes still need to be run by the stakeholders. Beth Potts was going to reach out to the stakeholders with these final parity questions, but I don't think that happened before she transitioned off. 

These should be run past the stakeholders before EDU parity is closed out.

**Comment by Karak888** [comment]
@jenniferlee-dsva to connect with @peggygannon to decide what pages need to be sent to the stakeholders.

**Comment by peggygannon** [comment]
Hello, all, I've updated this ticket with: 

--URLs for the teamsite/legacy pages (added to the list above)

--Location of the original EDU parity decks that Beth worked on: 
https://github.com/department-of-veterans-affairs/vets.gov-content/tree/master/content-topics/education/collateral/parity-copy-deck

**Comment by jenniferlee-dsva** [comment]
Per this note from Beth's old ticket - live Edu pages have been okayed and communicated to Edu stakeholders: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15111#issuecomment-467600261

**Comment by Karak888** [comment]
@AnneHurley and @RLHecht will make recommendatins in the ticket and then Randi and Anne will get them into Drupal.  Once that is done, @AnneHurley will send sMEs a heads-up to review these small edits.

**Comment by AnneHurley** [comment]
Creating a separate doc with all the recs above to be as clear as possible.

**Comment by AnneHurley** [comment]
[Education remaining issues.docx]
(https://github.com/department-of-veterans-affairs/va.gov-team/files/3752727/Education.remaining.issues.docx)
@jenniferlee-dsva  @RLHecht  
I've created a word doc listing all these missing pieces of info, with sections to be added, and where I think they would make sense, but please let me know if you disagree. 
A few notes:
1. The special Air Force requirements question puzzled me - I added in a section from legacy site but I am thinking perhaps the conditions are the same for those in all branches of the military now? And if that isn't correct and Air Force does need its own section of criteria, I would think we would need/want to add some of the other bullets, like having started an account, having a discharge other than dishonorable, etc. Thoughts?
2. The page that lets people go to their regional center to check on decision times by region is a really strange page: https://www.benefits.va.gov/REPORTS/detailed_claims_data.asp  *I don't see anywhere on that page where you can check decision times based on the three regional centers. And secondly, if we are going to refer people to check on their regional center, it would be helpful to have a map to show which areas of the country are covered by which - Buffalo, Muskogee, or St. Louis. 
3. A few things seem like they might have been intentionally omitted, so @jenniferlee-dsva  perhaps you could weigh in? 
4. I'm sure my stab at Plain Language-ing the children/spouses of disabled Veterans can use some help - I'm unclear on the "effective 2006" language, for instance. 
5. Note there is a date on one of our live pages that has a program expiring in November 2019.

Looking forward to feedback, thx.

**Comment by Karak888** [comment]
@AnneHurley to review and provide updates to @jenniferlee-dsva

**Comment by AnneHurley** [comment]
@RLHecht  and fyi @jenniferlee-dsva 
[Education parity edits for Drupal]
(https://app.zenhub.com/files/133843125/849656e3-ba97-473e-bc07-6f5cf1861899/download)

This doc contains the edits we decided were straightforward enough to just do straight in Drupal. There are three edits/pages in this batch. @RLHecht let me know if you have any questions.

**Comment by RLHecht** [comment]
@anne and @jenniferlee-dsva I've made these edits in Drupal. @peggygannon Can you please review them and set the pages to publish? They are Nodes 948, 922 and 940.

**Comment by peggygannon** [comment]
@RLHecht Can you add the names of the corresponding page titles for the nodes?

**Comment by jenniferlee-dsva** [comment]
Thanks all. - So to confirm @AnneHurley - these 3 pages have been updated and published? (Please be sure to let Mikki know for her Edu redirect epic.)

1/NON-COLLEGE DEGREE PROGRAMS
VA.gov page: www.va.gov/education/about-gi-bill-benefits/how-to-use-benefits/non-college-degree-programs/
legacy page: www.benefits.va.gov/gibill/non_college_degree.asp
[published]

2/CHOOSING A SCHOOL
VA.gov page: www.va.gov/education/choosing-a-school/
Legacy page: www.benefits.va.gov/gibill/choosing_a_school.asp
[published]

3/VEAP
Va.gov page: www.va.gov/education/other-va-education-benefits/veap/
Legacy page: www.benefits.va.gov/gibill/veap.asp
[published]

And these pages below - could you provide a status for these @AnneHurley ? (I.e., do you need me to connect you to Edu SME to walk through/sync on these? Just let me know if there's something I need to do for you here.)

VA.gov: www.va.gov/education/about-gi-bill-benefits/how-to-use-benefits/test-fees/
Legacy page: www.benefits.va.gov/gibill/licensing_certification.asp
Original comment: Add the below info to VA.gov page?
Fees connected with obtaining a license or certification are not reimbursable.
Review: Verification needed that content has been updated per the above statement. According to the website, the above statement isn't entirely true as certain license and certification fees are reimbursable. There is a URL link (WEAMS) to where a user can find covered licenses and certifications and the reimbursed amount.

VA.gov page: www.va.gov/education/how-to-apply/
Legacy page: www.benefits.va.gov/gibill/apply.asp
Original comment: Add a link to VA's interactive map for checking local processing times?
Review: Link to VA's interactive map for checking local processing has not been added.

Va.gov page: www.va.gov/education/other-va-education-benefits/reap/
Legacy page: www.benefits.va.gov/gibill/reap.asp
Original comment: Add information about Colmery Act and reservists crediting their REAP eligibility toward Post-9/11 benefits?
Review: Information above has not been added.

Va.gov page: www.va.gov/education/survivor-dependent-benefits/dependents-education-assistance/
Legacy page: www.benefits.va.gov/GIBILL/DEA.asp
Original comment: Add Additional Assistance information to the VA.gov page?
Review: Information above has not been added.

**Comment by AnneHurley** [comment]
[Education parity updates for SMEs]
(https://app.zenhub.com/files/133843125/23009246-fd18-4824-8008-25190918caed/download)
@RLHecht  This is ready for copyedit, let me know if you have any questions.
@jenniferlee-dsva @mnorthuis FYI - See my recommendations in the doc and let me know if you have other thoughts. And then Jen once it's been edited and you all approve, it's ready to go to the Education SMEs for their review.

**Comment by RLHecht** [comment]
@AnneHurley Here are my copy edits. I had one question I put in the comments
[Education parity-Randi edit.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/3773690/Education.parity-Randi.edit.docx)
 and just a few tweaks.

**Comment by jenniferlee-dsva** [comment]
Hi @AnneHurley - agree with your recommendations and Randi's edits ^. These are looking really good!

**Comment by RLHecht** [comment]
Just confirming that all these Education pages are in Drupal. Once Anne sends the copy deck to SMEs and gets their feedback I can input changes.

**Comment by AnneHurley** [comment]
[Education.parity-for SME review - v1]
(https://github.com/department-of-veterans-affairs/va.gov-team/files/3794905/Education.parity-for.SME.review.-.v1.docx)
@jenniferlee-dsva  This is ready for you to review. These are three outstanding Education parity edits we need to run by the SMEs. The third one is a confusing user flow (the map link that doesn't go to a map page but instead to that confusing "loads" page) so instead of trying to capture screen shots which I thought could be even more confusing I tried to walk them through the flows. And I can demonstrate via shared screen with them on a call if desired.

Let me know if you have any questions/edits.
@Karak888  and @RLHecht for visibility

**Comment by Karak888** [comment]
consolidate a few yellow ribbon pages in copy deck before sending to SME

**Comment by Karak888** [comment]
@AnneHurley can you please let me know if this has gone through SME spproval yet with the yellow ribbon pages.  Please let me know since the other pages are in drupal was hoping to close, but please let me know status if you can.  Thanks

**Comment by AnneHurley** [comment]
[Education parity-rewriting updates-SME approved]
(https://github.com/department-of-veterans-affairs/va.gov-team/files/3821223/Education.parity-rewriting.updates.-.v1.2.docx)

@RLHecht Note - we will continue to link to Team Site legacy state pages for Yellow Ribbon program because the updates are extensive each year.

One more piece of this: Anne to audit all state links to make sure they are all linking to the 2019-2020 school year. Will include list below.

**Comment by AnneHurley** [comment]
@RLHecht  @Karak888  @jenniferlee-dsva 
All the links on the Team Site Yellow Ribbon Schools are current, covering the 2019-2020 year. Most or all of the links on the Heroku page ^ above are outdated with 2018 in their URLs because they were built earlier this year before Team Site / Tammy had the current school year updates. 

I suggest the easiest thing to do is, when posting a link to a state URL, to simply change each instance of 2018 to 2019, because all the 2019 URLs are live.

**Comment by AnneHurley** [comment]
@peggygannon  @mnorthuis  @jenniferlee-dsva  @Karak888 --
Jen is concerned that having Yellow Ribbon Schools as a separate page will introduce possible confusion, because it can't live "under" the main Yellow Ribbon page because the latter is already three levels/clicks down. The options are:
1. Continue with Yellow Ribbon schools as a separate page, and having it be a "parallel" page to the main page in the IA/left nav, under Post-911 GI Bill. Pros: It visually gives user "space" to explore the schools via search or comparison tool to check schools in a relatively uncluttered experience. Cons: The navigation could be confusing because it would have to live as a "parallel" page to what is actually its "parent" page.
2. Incorporate all the information on the Yellow Ribbon participating schools page into the main Yellow Ribbon page, https://www.va.gov/education/about-gi-bill-benefits/post-9-11/yellow-ribbon-program/. Pros: No confusing IA in having the separate page be parallel to parent page since it would be all one page, and another pro is having all Yellow Ribbon info on the single page. Con: It would make for an extremely long single page; could the school search get lost potentially?

Were there discussions about these things in the spring, and if not should we discuss them now?

**Comment by jenniferlee-dsva** [comment]
Clarification:
<<Jen is concerned that having Yellow Ribbon Schools as a separate page will introduce possible confusion, because it can't live "under" the main Yellow Ribbon page because the latter is already three levels/clicks down.>>

Not really a confusion issue, but thinking about near future state of flattened nav IA (which won't allow child pages below L3). 

Since a) we are not going to migrate the 50 individual state table pages into the hub and b) only bring over the list of links pointing to each state table (on benefits.va.gov), I propose incorporating that links content and contextual intro copy as a new section on our current live Yellow Ribbon program page. Big caveat here being - could it be done in a way that makes sense and not difficult to find?  

Few ideas is to use our "On this page" jump links at the top like we do here:  http://preview-prod.vfs.va.gov/preview?nodeId=1013 

+ The current H2s under the "Learn more..." section are actually subtopics of the "Learn more..." H2 and should be H3s. 

+ That ^ means, that we might be able to add another H2 after or before the "Learn more..." H2 for participating schools -- like... 'Find/look up what schools offer Yellow Ribbon funding for tuition matching' or whatever. [And if needed using accordions for states by alpha groups?]

+ I think the 'What benefits can I get through...' H2 content is actually duplicative of the intro, and actually more specific and clearer, so there might be an opportunity there as well to eliminate this H2 and simply incorporate this language into the intro text or replace some of it. 

This would be easier visually, so I'll upload an annotated screenshot later. 

Longer term solution is to have a sortable/filterable table component in Drupal that we can migrate the state table info into, as its own 'tool' page. 

@mnorthuis  - leave you to recommend whether  it would be better in terms of IA:  
- to create a separate participating schools page now (that later  - no available time frame yet - turns into a table page; 
- or to combine this info with the main YRP info for now, since we don't have that dynamic table solution

**Comment by RLHecht** [comment]
@AnneHurley I made the SME changes to the two education pages in Drupal ("Survivors' and Dependents' Educational Assistance" and "Get paid back for test fees"). Per our conversation you are going to move the Yellow Ribbon changes into the other ticket.

**Comment by mnorthuis** [comment]
@jenniferlee-dsva The YR participating schools content could be moved into more resources if you feel it is too much to add on the existing YR page and the long term plan will be to migrate the content over.  

That said, I would prefer not to create a page that just has a few links that jump off to the legacy pages - that's just creating another step/click in the flow.   The [legacy YR](https://www.benefits.va.gov/gibill/yellow_ribbon.asp) page has a section of links at the bottom, and they just add 1 link each year to go to the new pages.  This seems like it would be the simplest way to handle this until we have a better approach for Tier 2/More Resources content.

**Comment by AnneHurley** [comment]
[Final SME edits to 2 Education pages]
(https://app.zenhub.com/files/133843125/a3b5cd05-42d4-43c8-9a6e-1b6add83dd90/download)
@RLHecht  - Small fixes to both the pages from the SMEs, attached. Happy to screen share to answer the open ended question they ask with you. There's another q about a link which I'm sure will work when the page is pushed from Prod to live, yes?

**Comment by RLHecht** [comment]
@AnneHurley These changes have all been made and I published the two pages after you reviewed with the screen share. Should this ticket be moved to validate and then closed?

**Comment by AnneHurley** [comment]
@RLHecht  Yes, thanks. When the pages are live I'll send them to the SMEs to close the loop. Thanks much!

**Comment by Karak888** [comment]
so @RLHecht are we good to go with the yellow ribbon to close our or are we still awaiting something? Thanks!

**Comment by RLHecht** [comment]
@Karak888 I commented in [this ticket](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/2612) that Peggy or Jen need to review and publish the Yellow Ribbon page.But if this ticket is for the other two pages (which I think it is?), those pages are now live and we can close. @AnneHurley, can you confirm?

**Comment by AnneHurley** [comment]
@Karak888  and @RLHecht yes if the ^ pages are live, we can close this one.

**Comment by Karak888** [comment]
@RLHecht @AnneHurley Since the 2 pages above becAuse we have another ticket specific to Yellow Ribbon and this was put as a blocker (no longer blocker), I am going to close this ticket out and we'll track Yellow Ribbon separately.  :CC @jenniferlee-dsva

**Comment by mnorthuis** [comment]
@Karak888 @RLHecht Can you please clarify what two pages have changes that are now live from the education parity?  I'm struggling to  understand which 2 are being referenced here, there's a lot of pages mentioned in this ticket.

**Comment by AnneHurley** [comment]
@mnorthuis @Karak888 @RLHecht  The two non-Yellow Ribbon pages we made edits to are:

1. https://www.va.gov/education/about-gi-bill-benefits/how-to-use-benefits/test-fees/ to better set up the L&C tool after the link
2. https://www.va.gov/education/survivor-dependent-benefits/dependents-education-assistance/ to add new language and links about vocational education and rehabilitation education.
Let me know if you have further questions.

**Comment by jenniferlee-dsva** [comment]
As part of every content parity/legacy rewrite or updating, please create a subtask to retire the legacy pages that have been fully 'migrated' into our hub. @AnneHurley @Karak888  - please let @lunascoop know when the redirects have been implemented, so she can add them to the consolidated redirect spreadsheet she's creating.

This is a messy process right now, so we'll figure out something more streamlined for later content tickets with a redirect step.
